seacrowd/sea_datasets/indoler/indoler.py

```diff
@@ -0,0 +1,225 @@
+  doi={10.1109/ICACSIS51025.2020.9263157}}
+"""
+
+_DATASETNAME = "indoler"
+
+_DESCRIPTION = """\
+https://github.com/ir-nlp-csui/indoler/tree/main
+The data can be used for NER Task in legal documents.
+The dataset contains 993 annotated court decission document. 
+The document was taken from Decision of the Supreme Court of Indonesia. 
+The documents have also been tokenized and cleaned
+"""
+
+_HOMEPAGE = "https://github.com/ir-nlp-csui/indoler/tree/main"
+
+_LANGUAGES = ['ind']  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+
+_LICENSE = Licenses.UNKNOWN.value 
+
+_LOCAL = False
+
+_URLS = {
+    _DATASETNAME: {
+        "test_idx": "https://raw.githubusercontent.com/ir-nlp-csui/indoler/main/test.ids.csv",
+        "train_idx": "https://raw.githubusercontent.com/ir-nlp-csui/indoler/main/train.ids.csv",
+        "valid_idx": "https://raw.githubusercontent.com/ir-nlp-csui/indoler/main/val.ids.csv",
+        "full_data": "https://raw.githubusercontent.com/ir-nlp-csui/indoler/main/data.json"
+    },
+}
+
+_SUPPORTED_TASKS = [Tasks.NAMED_ENTITY_RECOGNITION]  
+
+_SOURCE_VERSION = "2.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+
+class IndoLer(datasets.GeneratorBasedBuilder):
+    """https://github.com/ir-nlp-csui/indoler/tree/main
+The data can be used for NER Task in legal documents
+The dataset contains 993 annotated court decission document. 
+The document was taken from Decision of the Supreme Court of Indonesia. 
+The documents have also been tokenized and cleaned"""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name="indoler_source",
+            version=SOURCE_VERSION,
+            description="indoler source schema",
+            schema="source",
+            subset_id="indoler",
+        ),
+        SEACrowdConfig(
+            name="indoler_seacrowd_seq_label",
+            version=SEACROWD_VERSION,
+            description="indoler SEACrowd schema",
+            schema="seacrowd_seq_label",
+            subset_id="indoler",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "indoler_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+
+        if self.config.schema == "source":
+            features = datasets.Features({
+                "id": datasets.Value("string"),
+                "owner": datasets.Value("string"),
+                "lawyer": datasets.ClassLabel(names=[False, True]),
+                "verdict": datasets.ClassLabel(names=["guilty", "bebas", "lepas"]),
+                "indictment": datasets.ClassLabel(names=["NA", "tunggal", "subsider", "komul", "alternatif", "kombinasi", "gabungan"]),
+                "text-tags": datasets.Sequence(datasets.ClassLabel(names=
+                    ['O', 'B-Jenis Amar', 'B-Jenis Dakwaan', 'B-Jenis Perkara', 'B-Melanggar UU (Dakwaan)', 
```

Comment:
Could we use a constant instead? Example:
```
NAMED_ENTITIES = ['O', 'B-Jenis Amar', 'B-Jenis Dakwaan', 'B-Jenis Perkara', 'B-Melanggar UU (Dakwaan)', 
                    'B-Melanggar UU (Pertimbangan Hukum)', 'B-Melanggar UU (Tuntutan)', 'B-Nama Hakim Anggota', 'B-Nama Hakim Ketua', 
                    'B-Nama Jaksa', 'B-Nama Panitera', 'B-Nama Pengacara', 'B-Nama Pengadilan', 
                    'B-Nama Saksi', 'B-Nama Terdakwa', 'B-Nomor Putusan', 'B-Putusan Hukuman', 
                    'B-Tanggal Kejadian', 'B-Tanggal Putusan', 'B-Tingkat Kasus', 'B-Tuntutan Hukuman', 
                    'I-Jenis Amar', 'I-Jenis Dakwaan', 'I-Jenis Perkara', 'I-Melanggar UU (Dakwaan)', 
                    'I-Melanggar UU (Pertimbangan Hukum)', 'I-Melanggar UU (Tuntutan)', 'I-Nama Hakim Anggota', 'I-Nama Hakim Ketua', 
                    'I-Nama Jaksa', 'I-Nama Panitera', 'I-Nama Pengacara', 'I-Nama Pengadilan', 
                    'I-Nama Saksi', 'I-Nama Terdakwa', 'I-Nomor Putusan', 'I-Putusan Hukuman', 
                    'I-Tanggal Kejadian', 'I-Tanggal Putusan', 'I-Tingkat Kasus', 'I-Tuntutan Hukuman']
...
"text-tags": datasets.Sequence(datasets.ClassLabel(names=NAMED_ENTITIES)),
```

Reply:
Implemented!